tests/utils.py

```diff
@@ -333,6 +358,7 @@ def __init__(
         self.timeout = datetime.timedelta(0, timeout)
         self.daemon = daemon
         self.method = method
+        self.min_gpu_memory = min_gpu_memory / self.nproc_per_node
```

Comment:
remove `os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"` according to the discussions cc @Nic-Ma